File: Level 0/코딩 기초 트레이닝/간단한 식 계산하기/solution.js
const operators = {
  "+": (a, b) => a + b,
  "-": (a, b) => a - b,
  "*": (a, b) => a * b,
};

function solution(binomial) {
  const [a, operator, b] = binomial.split(" ");
  return operators[operator](+a, +b);
}

File: Level 0/코딩 기초 트레이닝/간단한 식 계산하기/solution.py
operators = {
    "+": lambda a, b: a + b,
    "-": lambda a, b: a - b,
    "*": lambda a, b: a * b,
}


def solution(binomial):
    a, operator, b = binomial.split(" ")
    return operators[operator](int(a), int(b))
